eslint-plugin-expensify/onyx-props-must-have-default.js

```diff
@@ -0,0 +1,248 @@
+                makeReport(node, MUST_USE_VARIABLE_FOR_ASSIGNMENT, key);
+                return 1;
+            }
+            return 0;
+        }
+
+        return {
+            CallExpression(node) {
+                const name = lodashGet(node, 'callee.name');
+                if (!name) {
+                    return;
+                }
+
+                if (name !== 'withOnyx') {
+                    return;
+                }
+
+                if (node.arguments.length === 0) {
+                    context.report({
+                        node,
+                        message: ONYX_ONE_PARAM,
+                    });
+                }
+
+                // Get all the tree ancestors
+                const ancestors = context.getAncestors();
+
+                // Check the component is an HOC that wraps a component
+                const wrappedComponent = _.find(
+                    ancestors,
+                    n => lodashGet(n, 'type') === 'ExportDefaultDeclaration'
+                        && lodashGet(n, 'declaration.params[0].name') === 'WrappedComponent',
+                );
+
+                let propTypesDeclaration; let
+                    defaultPropTypesDeclaration;
+                if (wrappedComponent) {
+                    propTypesDeclaration = getPropsDeclaration(wrappedComponent.declaration.body.body, 'propTypes');
+                    defaultPropTypesDeclaration = getPropsDeclaration(wrappedComponent.declaration.body.body, 'defaultProps');
+                } else {
+                    propTypesDeclaration = getPropsDeclaration(ancestors[0].body, 'propTypes');
+                    defaultPropTypesDeclaration = getPropsDeclaration(ancestors[0].body, 'defaultProps');
+                }
+
+                if (isDirectAssignment(node, propTypesDeclaration, 'propTypes') || isDirectAssignment(node, defaultPropTypesDeclaration, 'defaultProps')) {
+                    return;
+                }
+
+                const scope = context.getScope();
+
+                // Get the assigned variable name
+                const propTypesVar = lodashGet(propTypesDeclaration, 'expression.right.name');
+                const defaultPropsVar = lodashGet(defaultPropTypesDeclaration, 'expression.right.name');
+
+                // Get the list of properties
+                const propTypesProperties = getPropsValue(scope, propTypesVar);
+                const defaultPropsProperties = getPropsValue(scope, defaultPropsVar);
+
+                if (!propTypesProperties) {
+                    makeReport(node, HAVE_PROP_TYPES);
+                    return;
+                } if (!defaultPropsProperties) {
+                    makeReport(node, HAVE_DEFAULT_PROPS);
+                    return;
+                }
+
+                // Get the list of properties of withOnyx
+                const onyxProperties = lodashGet(node, 'arguments[0].properties');
+
+                _.forEach(onyxProperties, (property) => {
```

Comment:
Isn't `_.forEach` is not available on the _underscore.js_? We can replace it with `_.each`.

Reply:
@Prince-Mendiratta Thanks! I don't see the _alias_ note. I'm inclined to use `_.each` instead, it consistent with the current usage and avoiding the confusion too.